currency_converter.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/python3
+
+import argparse
+import json
+import sys
+
+from forex_python.converter import CurrencyCodes
+from forex_python.converter import CurrencyRates
+from forex_python.converter import RatesNotAvailableError
+
+
+code_dictionary = \
+    { u'$':   'USD'
+    , u'Fr.': 'CHF'
+    , u'Ft':  'HUF'
+    , u'HK$': 'HKD'
+    , u'Kr':  'DKK'
+    , u'Kč':  'CZK'
+    , u'L':   'RON'
+    , u'NZ$': 'NZD'
+    , u'R$':  'BRL'
+    , u'R':   'RUB'
+    , u'RM':  'MYR'
+    , u'Rp':  'IDR'
+    , u'S$':  'SGD'
+    , u'W':   'KRW'
+    , u'kn':  'HRK'
+    , u'kr':  'NOK'
+    , u'zł':  'PLN'
+    , u'£':   'GBP'
+    , u'¥':   'JPY'
+    , u'฿':   'THB'
+    , u'₪':   'ILS'
+    , u'€':   'EUR'
+    , u'₱':   'PHP'
+    , u'₹':   'INR'
+    }
+
+def main(argv):
+
+    parser = argparse.ArgumentParser(description='Currency converter.')
+    # Options:
```

Comment:
I think really not necessary doc

`parser.add_argument("--amount"` is readable enough